builtin/providers/aws/resource_aws_cloudwatch_event_target.go

```diff
@@ -203,10 +243,59 @@ func buildPutTargetInputStruct(d *schema.ResourceData) *events.PutTargetsInput {
 		e.InputPath = aws.String(v.(string))
 	}
 
+	if v, ok := d.GetOk("role_arn"); ok {
+		e.RoleArn = aws.String(v.(string))
+	}
+
+	if v, ok := d.GetOk("run_command_targets"); ok {
+		params, err := expandAwsCloudWatchEventTargetRunParameters(v.([]interface{}))
+		if err != nil {
+			return nil, err
+		}
+
+		e.RunCommandParameters = params
+	}
+
 	input := events.PutTargetsInput{
 		Rule:    aws.String(d.Get("rule").(string)),
 		Targets: []*events.Target{e},
 	}
 
-	return &input
+	return &input, nil
+}
+
+func expandAwsCloudWatchEventTargetRunParameters(config []interface{}) (*events.RunCommandParameters, error) {
```

Comment:
I don't see any errors that could be returned in this function. Could potentially error case each interface cast and return those errors if found, or remove the error return from the function? 
If the error is removed from the function as a return parameter, could also remove it from the `buildPutTargetInputStruct()` function that calls this function.